app/HutchAgent/Services/ActionHandlers/FetchAndExecuteActionHandler.cs

```diff
@@ -99,14 +104,24 @@ public async Task HandleAction(string jobId, object? payload)
       throw new InvalidOperationException(
         $"Hutch somehow queued an Action for a Job (id: {jobId}) that doesn't exist.", e);
     }
+    catch (InvalidDataException e)
```

Comment:
don't capture the exception into a variable if you're not gonna use it.
```suggestion
    catch (InvalidDataException)
```